ios/Classes/SwiftFlutterVlcPlayerPlugin.swift

```diff
@@ -147,6 +147,8 @@ public class VLCView: NSObject, FlutterPlatformView {
                     result(FlutterMethodNotImplemented)
                     return
                 }
+            } else {
+                print("no arguments")
```

Comment:
We usually try to avoid print statements.

Reply:
Debug Log statement may be better